modules/nf-core/pindel/pindel/main.nf

```diff
@@ -0,0 +1,50 @@
+process PINDEL_PINDEL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::pindel=0.2.5b9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pindel:0.2.5b9--h06e5f0a_6':
+        'quay.io/biocontainers/pindel:0.2.5b9--h06e5f0a_6' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), val(insert)
```

Comment:
Although when looking further at your code, it may one of those magical exceptions to the rule, but you should post about it on slack in the `#modules` channel. That's were this gets decided